x-pack/legacy/plugins/uptime/public/components/functional/snapshot.tsx

```diff
@@ -4,15 +4,17 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiStat, EuiTitle } from '@elastic/eui';
-import { EuiSpacer } from '@elastic/eui';
-import { i18n } from '@kbn/i18n';
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
 import React from 'react';
+import { DonutChart } from './charts';
 import { Snapshot as SnapshotType } from '../../../common/graphql/types';
 import { UptimeGraphQLQueryProps, withUptimeGraphQL } from '../higher_order';
 import { snapshotQuery } from '../../queries';
-import { SnapshotLoading } from './snapshot_loading';
+import { ChartWrapper } from './charts/chart_wrapper';
+
+const SNAPSHOT_CHART_WIDTH = 128;
+const SNAPSHOT_CHART_HEIGHT = 128;
```

Comment:
`144px` for width and height of the chart seem perfect to me.
<img width="308" alt="Screenshot 2019-10-09 at 10 52 42" src="https://user-images.githubusercontent.com/20965076/66466473-0eb6ff00-ea83-11e9-96e8-d1106ba353a1.png">

Reply:
Fixed in https://github.com/elastic/kibana/pull/47176/commits/8c65f72f90a1af07381c4a7e05044958f4c22214.